src/bots/coverbot/index.ts

```diff
@@ -443,11 +467,11 @@ export class Coverbot implements Bot {
       this.relayTimeouts.delete(relayerAddress)
 
       // 4.
-      const score = await this._getHoprAddressScore(relayerAddress)
-      const newScore = score + ScoreRewards.relayed
-
-      await Promise.all([
-        this._setHoprAddressScore(relayerAddress, newScore),
+      const [ newScore ] = await Promise.all([
+        this._increaseHoprAddressScore(relayerAddress, (prevScore) => {
+          if (!prevScore) return ScoreRewards.relayed
```

Comment:
here I decided to award the points even though the previous score should be `100`, my thinking is that awarding points shouldn't depend if the first `100` points were awarded

Reply:
Should be fine, let me go ahead and test it myself